docs/eventing/channels/subscriptions.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Subscriptions"
+weight: 100
+type: "docs"
+showlandingtoc: "false"
+---
+
+After you have created a channel and a sink, you can create a subscription to enable event delivery.
+
+## Creating a subscription
+
+{{< tabs name="Creating a subscription" default="kn" >}}
+{{% tab name="kn" %}}
+
+Create a subscription between a channel and a sink:
+
+```
+kn subscription create <subscription_name> \
+  --channel <Group:Version:Kind>:<channel_name> \
+  --sink <sink_prefix>:<sink_name> \
+  --sink-reply <sink_prefix>:<sink_name> \
+  --sink-dead-letter <sink_prefix>:<sink_name>
+```
+
```

Comment:
I think we should either explain all options (like `--sink-reply` and `--sink-dead-letter`) and also the naming convention here, i.e. `<sink_prefix>`, or reduce the example to the minimum required.

Let me try to give some raw input here, that we could use here to explain:

* `--channel` is the source for the cloud-events that should be processed. You have to provide the channel name here. If you don't use a default channel (backed by the `Channel` resource), you have to prefix the channel name with the Group/Version/Kind coordinates of the typed channel, e.g. "
" for a Kafka backed channel.

Reply:
> We could also split off the description of the sink format into a separate document as it can be reused when describing the --sink option for source and triggers.

Let's open a new issue for this / follow up in a separate PR. I'm wary of having too much scope creep in this one.